releasenotes/notes/deprecate-pulse-instruction-call-52dca0dd26e1c768.yaml

```
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Removed the deprecated class :class:`qiskit.pulse.instructions.Call`. Use function
+    :func:`qiskit.pulse.builder.call` instead.
```

```suggestion
    Removed the deprecated class :class:`qiskit.pulse.instructions.Call`. 
    This is no longer necessary and no alternative pulse instruction is provided.
```

The `Call` instruction used to be a wrapper of `Schedule` so that `ScheduleBlock` can call `Schedule` program (device calibration) in the builder context as a subroutine. This was needed to handle `Schedule` like an instruction, but now `Schedule` injected into `ScheduleBlock` context is implicitly converted into `ScheduleBlock` and flattened. So this instruction is just no longer necessary.